rules/jboss-eap/weblogic/WebLogicWebServiceRules.windup.xml

```diff
@@ -129,6 +130,33 @@
                 </iteration>
             </perform>
         </rule>
+        <rule id="WebLogicWebServiceRules_36ojlq">
+            <when>
+                <javaclass references="weblogic.common.T3ServicesDef" as="default">
+                    <location>TYPE</location>
+                </javaclass>
+            </when>
+            <perform>
+                <iteration>
+                    <hint message="In JBoss 5, look up the MBean replacement for the T3ServicesDef." effort="1">
+                    </hint>
+                </iteration>
+            </perform>
+        </rule>
+        <rule id="WebLogicWebServiceRules_38jbae">
+            <when>
+                <javaclass references="weblogic.common.T3ServicesDef" as="default">
+                    <location>TYPE</location>
+                </javaclass>
+            </when>
+            <perform>
+                <iteration>
+                    <hint message="In JBoss 6, inject the @Singleton, @Startup EJB3." effort="1">
```

Comment:
This part of the rule update shouldn't be here at all. It's updating the old rules by mistake. It's probably a mistake while rebasing... I'll remove it.